### PR TITLE
hotel endpoint document update

### PR DESCRIPTION
**Pull request by runqiaolin**
!!!!! STOP AND READ !!!!!

If the dropdown above says "base fork: lord/master", you are submitting your change to ALL USERS OF SLATE, not just your company. This is probably not what you want. Click "base fork" to change it to the right place.

If you're actually trying to submit a change to upstream Slate, please submit to our dev branch, PRs sent to the master branch are generally rejected.

### CODE REVIEW

File: source/includes/_hotels.md [hunk 1]
```diff
@@ -52,10 +52,16 @@ The following URL shows how the search prices endpoint can be used to retrieve p
 
 *API endpoint*
 
+B2B partner please use v1 endpoint
+
 `GET /v1/prices/search/entity/{entity_id}`
 
+v3 is new endpoint, support dbook booking and cug override
```
Comment (williamdong81) [inline source/includes/_hotels.md]: I'll prefer to describe it as "V3 is a new endpoint which supports direct booking and cug overriding."

File: source/includes/_hotels.md [hunk 2]
```diff
@@ -92,14 +98,19 @@ GET "https://www.skyscanner.net/g/hotels/v1/prices/search/entity/{entity_id}
 | `rooms` <br><span class="required">REQUIRED</span> | Number of rooms<br>default: 1 |
 | `adults` <br><span class="required">REQUIRED</span> | Number of adults<br>default: 2 |
 | `apikey` <br><span class="required">REQUIRED</span> | This parameter is required to be on every single request any client does |
-| `images` <br><span class="optional">OPTIONAL</span> | Maximum number of images to retrieve per each hotel<br>between 1-30, default: 3 |
+| `images` <br><span class="optional">OPTIONAL</span> | Maximum number of images to retrieve per each hotel<br>between 1-30, default: 1 |
```
Comment (williamdong81) [inline source/includes/_hotels.md]: In general, per means every, similar meaning with each, here I suggest to write as "retrieve for each hotel".

File: source/includes/_hotels.md [hunk 3]
```diff
@@ -92,14 +98,19 @@ GET "https://www.skyscanner.net/g/hotels/v1/prices/search/entity/{entity_id}
...
 | `price_min` <br><span class="optional">OPTIONAL</span> | Filter. Return only hotels where the cheaper price is at least price_min (included). Cannot be used together with price_buckets |
 | `price_max` <br><span class="optional">OPTIONAL</span> | Filter. Return only hotels where the cheaper price is at most price_max (included). Cannot be used together with price_buckets |
 | `price_buckets` <br><span class="optional">OPTIONAL</span> | OR filter. Return only hotels with offers inside the specified buckets. Cannot be used together with price_min/price_max |
+| `price_bucket_step` <br><span class="optional">OPTIONAL</span> | Fix step for the buckets. Need to be used together with price_buckets, default step will be dynamic |
+| `price_bucket_number` <br><span class="optional">OPTIONAL</span> | This parameter allows you to set the price buckets number, default: 5 |
+| `price_buckets_strategy` <br><span class="optional">OPTIONAL</span> | price_buckets_strategy is 2 means use the new price buckets thread (split most 30 buckets), others use old price bucket thread (split most price_bucket_number buckets) |
```
Comment (williamdong81) [inline source/includes/_hotels.md]: Just some small corrections here, means use -> means to use, split most xx buckets -> split into xx buckets at most.

File: source/includes/_hotels.md [hunk 4]
```diff
@@ -108,11 +119,21 @@ GET "https://www.skyscanner.net/g/hotels/v1/prices/search/entity/{entity_id}
...
+| `index` <br><span class="optional">OPTIONAL</span> | Mark the times of the same requests sent to bellboy, start from 1, default: 0 |
+| `sorting_version` <br><span class="optional">OPTIONAL</span> | (B2B partner need't set the value) Hotel sorting score calculation options<br>mr4: RelevanceContainerScoring,<br>mr5: IronRankScoring,<br>mr6: IronCatRankScoring,<br>mr4a: RelevanceContainerHotelModelScoring,<br>mr7: HotelBookingModelScoring,<br>mr8: IronTrainModelScoring,<br>default: mr4 |
```
Comment (williamdong81) [inline source/includes/_hotels.md]: Some suggestions to upgrade here, need't -> needn't.

Comment (williamdong81) [reply source/includes/_hotels.md]: Same for following same places.

File: source/includes/_hotels.md [hunk 5]
```diff
@@ -108,11 +119,21 @@ GET "https://www.skyscanner.net/g/hotels/v1/prices/search/entity/{entity_id}
...
+| `t_version` <br><span class="optional">OPTIONAL</span> | (B2B partner need't set the value) Used for ABTesting, default is '' |
+| `app_name` <br><span class="optional">OPTIONAL</span> | (B2B partner need't set the value) This parameter allows you to the app name, default is '' |
```
Comment (williamdong81) [inline source/includes/_hotels.md]: allows you to set the app name? Same as below points.

File: source/includes/_hotels.md [hunk 6]
```diff
@@ -760,8 +813,14 @@ Is a regular practice following the user flow first of all search for prices usi
 
 *API endpoint*
 
+B2B partner please use v1 endpoint
+
 `GET /v1/prices/hotel/{hotel_id}`
 
+v3 is new endpoint, support dbook booking and cug override
```
Comment (williamdong81) [inline source/includes/_hotels.md]: Same as above comment.

File: source/includes/_hotels.md [hunk 7]
```diff
@@ -799,146 +858,400 @@ GET "https://www.skyscanner.net/g/hotels/v1/prices/hotel/{hotel_id}
 | `rooms` <br><span class="required">REQUIRED</span> | Number of rooms<br>default: 1 |
 | `adults` <br><span class="required">REQUIRED</span> | Number of adults<br>default: 2 |
 | `apikey` <br><span class="required">REQUIRED</span> | This parameter is required to be on every single request any client does |
-| `images` <br><span class="optional">OPTIONAL</span> | Maximum number of images to retrieve per each hotel<br>between 1-30, default: 3 |
+| `images` <br><span class="optional">OPTIONAL</span> | Maximum number of images to retrieve per each hotel<br>between 1-30, default: 1 |
```
Comment (williamdong81) [inline source/includes/_hotels.md]: Same as above comment.

File: source/includes/_hotels.md [hunk 8]
```diff
@@ -799,146 +858,400 @@ GET "https://www.skyscanner.net/g/hotels/v1/prices/hotel/{hotel_id}
...
 | `entity_id` <br><span class="optional">OPTIONAL</span> | This field should be present when this endpoint has been called when the user has already looked for prices in an upper entity containing the hotel in order to reuse the cached prices.<br>For example: A user searches for prices in Paris (the service stores Paris prices). And then, the user opens a hotel details page. In this case, the request to the service must have the entity_id param fulfilled with the Paris entity_id in order to reuse the prices.|
+| `index` <br><span class="optional">OPTIONAL</span> | Mark the times of the same requests sent to bellboy, start from 1, default: 0 |
+| `t_version` <br><span class="optional">OPTIONAL</span> | (B2B partner need't set the value) Used for ABTesting, default is '' |
+| `app_name` <br><span class="optional">OPTIONAL</span> | (B2B partner need't set the value) This parameter allows you to the app name, default is '' |
```
Comment (williamdong81) [inline source/includes/_hotels.md]: Same as above comment.

**Comment by runqiaolin** [comment]
Thanks for the review, already updated as per the comments.

**Pull request merged**